src/database_utils/nerf_synthetic/train_test_creators/TrainTestCreator01_UniformSampling.py

```diff
@@ -1,124 +0,0 @@
-# Shree KRISHNAya Namaha
```

Comment:
Why did you delete this file?

Reply:
Added it back, vscode had too many Git changes so it was truncating.